app/presenters/document_collection_presenter.rb

```diff
@@ -52,6 +52,22 @@ def group_document_links(group, group_index)
     end
   end
 
+  def hide_public_updated_at(link)
```

Comment:
Just a style/readability comment: I'd expect a method called `hide_public_updated_at(link)` to hide the public_update_at field, but this method is more often than not going to be returning a public updated_at field right? Maybe `group_document_link_public_updated_at` is more informative? And it should be a private method to follow the existing convention in this class.